docs-source/source/contribute.rst

```diff
@@ -105,6 +105,69 @@ before developing and algorithm are:
         * List of floats: ``List[float]``
         * Optional type: ``Optional[float]``
 
+Functions exposed to Charts should also contain the following: 
```

Comment:
What is the difference between https://indsl.docs.cognite.com/contribute.html#document-your-algorithm and the section you have added? 
Instead of creating a new section, could we just update the old section with more information? Maybe improve the formatting a bit there as well?